### DISCUSSION TITLE
特殊字符显示不了，比如爱心♡，如果去掉layui.css引用，可以正常显示，就无法使用layui的布局等渲染了。请问怎么解决呢？

### DISCUSSION THREAD

**Discussion by wangwenjie7** [layui/layui#1639] [问答] [answered]
特殊字符显示不了，比如爱心♡，显示的是三条横线，如果去掉layui.css引用link( <link rel="stylesheet" type="text/css" href="/static/layui/css/layui.css"> )，可以正常显示，就无法使用layui的布局等渲染了。请问怎么解决呢?

**Comment by Sight-wcg** [comment, marked as answer]
```
body{
 font-family: 'Segoe UI Emoji',  'Segoe UI Symbol';
}
```

**Reply by wangwenjie7** [reply]
谢谢啊，试了这个'Segoe UI Symbol'字体就是我想要得样子，前端小白在此太感谢了啊♡♡♡♡♡♡♡
